src/main/java/Task_25/Main.java

```diff
@@ -0,0 +1,103 @@
+package Task_25;
+
+import java.sql.*;
+
+public class Main {
+
+    public static MySQLDriverManager driverManager = MySQLDriverManager.getInstance();
+    public static Connection connection;
+    public static Statement statement;
+    public static ResultSet resultSet;
+
+    static {
+        try {
+            connection = driverManager.getConnection();
```

Comment:
Let me describe a bit, what I meant by the previous comment:
you can initialize `driverManager = MySQLDriverManager.getInstance();` here, in static block, but all other initializations, loke connection statement or resultSet should be placed exacttly in the methods, where they are needed (even if there will be some duplicate code).
the main idea is to make such methods independent